ansible/group_vars/all/openstack

```diff
@@ -21,6 +21,10 @@ openstack_routers: []
 # stackhpc.os-networks role.
 openstack_security_groups: []
 
+# List of RBAC definitions in the openstack projct. Format is as required by the
+# stackhpc.os-networks role.
+openstack_networks_rbac: []
```

Comment:
Could you also add it to etc/openstack-config/openstack-config.yml so that the option is visible?

Reply:
Done